lib/ripper-tags.rb

```diff
@@ -53,6 +54,9 @@ def self.option_parser(options)
           options.exclude << pattern
         end
       end
+      opts.on("--fields=+n", "Include line number information in the tag") do |value|
+        options.line_numbers = value == "+n"
```

Comment:
This is not going to work if someone passed `--fields=n` or `--fields=abcn` (where "abc" are flags that we currently don't support). Please copy or extract [the existing `--extra` mechanism](https://github.com/tmm1/ripper-tags/blob/a1e7f0c8edd0510739bd07815ff617578ffa8cc3/lib/ripper-tags.rb#L68-L77) for processing these flags and later just check if `n` was among them.